File: SEM5/PFCD/Assignment4/Q4.py
numbers = list(map(int, input("Enter numbers: ").split()))
element = int(input("Enter element to remove: "))

while element in numbers:
    numbers.remove(element)

print("Updated list:", numbers)

'''
Enter numbers: 2 3 4 5 3 78 3 9
Enter element to remove: 3
Updated list: [2, 4, 5, 78, 9]

'''

File: SEM5/PPWC/Minor Assignment4/Q4.c
#include <stdio.h>

int main() {
    int x = 89;
    int *p1 = &x;
    int *p2 = &x;
    int *p3 = &x;
    printf("Value of x using p1 = %d\n", *p1);
    *p3 = 100;
    printf("Updated value of x using p3 = %d\n", *p3);
    printf("Value of x using p1 = %d\n", *p1);
    return 0;
}

/*
Value of x using p1 = 89
Updated value of x using p3 = 100
Value of x using p1 = 100
*/